sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,741 @@
+## API Report File for "@azure/ai-form-recognizer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/core-http';
+import * as coreHttp from '@azure/core-http';
+import { HttpRequestBody } from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-http';
+import { Poller } from '@azure/core-lro';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/identity';
+import { WebResource } from '@azure/core-http';
+
+// @public (undocumented)
+export type AnalyzeFormResult = Omit<AnalyzeResult, "documentResults">;
+
+// @public
+export interface AnalyzeLayoutAsyncHeaders {
+    operationLocation: string;
+}
+
+// @public
+export type AnalyzeLayoutAsyncResponseModel = AnalyzeLayoutAsyncHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: AnalyzeLayoutAsyncHeaders;
+    };
+};
+
+// @public (undocumented)
+export type AnalyzeLayoutOperationResult = Omit<AnalyzeOperationResultModel, 'analyzeResult'> & {
```

Comment:
since type calculus gets complicated, I ended up re-writing the generated interface by hand in search to avoid these kinds of substitutions

Reply:
My usage is mostly: replace this field with that field that has a different type and/or different name.

Reply:
Fixed. Most type calculus should been replaced with explicit definitions.